# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# abuyop <abuyop@gmail.com>, 2017-2018
# Khairulanuar, 2013
# Mohd Shahril Bin Zainol Abidin <mohd_shahril_96@yahoo.com>, 2013
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-08-16 11:40+0200\n"
"PO-Revision-Date: 2018-08-16 14:46+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Malay (Malaysia) (http://www.transifex.com/otf/torproject/language/ms_MY/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ms_MY\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Anda mesti menjalankan aplikasi ini sebagai root"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr ""

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Tulis ke peranti dengan kelajuan %(speed)d MB/sec"

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Terdapat ralat ketika melakukan baris perintah berikut: `%(command)s`.\nSatu log ralat terperinci dihasilkan ke dalam '%(filename)s'."

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Mengesahkan hasil tambah semak SHA1 bagi imej CDLangsung..."

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Mengesahkan hasil tambah semak SHA256 bagi imej CDLangsung..."

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Ralat: SHA1 bagi CD Langsung anda tidak sah. Anda boleh menjalankan program ini dengan argumen --noverify untuk melepasi semakan pengesahan."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "ISO tidak diketahui, melangkau pengesahan hasil tambah semak"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr ""

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Mencipta %sMB lapisan berterusan"

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Tidak boleh menyalin %(infile)s ke %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "Membuang OS Langsung sedia ada"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:456
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Tidak boleh lakukan chmod %(file)s: %(message)s"

#: ../tails_installer/creator.py:449
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Tidak dapat membuang fail daripada OSLangsung terdahulu: %(message)s"

#: ../tails_installer/creator.py:462
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Tidak dapat membuang direktori daripada OSLangsung terdahulu: %(message)s"

#: ../tails_installer/creator.py:510
#, python-format
msgid "Cannot find device %s"
msgstr "Tidak dapat cari peranti %s"

#: ../tails_installer/creator.py:711
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Tidak dapat menulis ke %(device)s, melangkau."

#: ../tails_installer/creator.py:741
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr ""

#: ../tails_installer/creator.py:784 ../tails_installer/creator.py:1008
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Sistem fail tidak diketahui. Peranti anda mungkin harus diformatkan semula."

#: ../tails_installer/creator.py:787 ../tails_installer/creator.py:1011
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Sistem fail tidak disokong: %s"

#: ../tails_installer/creator.py:805
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:810
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Tidak boleh melekapkan peranti: %(message)s"

#: ../tails_installer/creator.py:815
msgid "No mount points found"
msgstr "Tiada titik lekap ditemui"

#: ../tails_installer/creator.py:826
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr ""

#: ../tails_installer/creator.py:836
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr ""

#: ../tails_installer/creator.py:840
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr ""

#: ../tails_installer/creator.py:851
#, python-format
msgid "Mount %s exists after unmounting"
msgstr ""

#: ../tails_installer/creator.py:864
#, python-format
msgid "Partitioning device %(device)s"
msgstr ""

#: ../tails_installer/creator.py:993
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr ""

#: ../tails_installer/creator.py:996
msgid "Trying to continue anyway."
msgstr "Cuba teruskan jua."

#: ../tails_installer/creator.py:1005 ../tails_installer/creator.py:1401
msgid "Verifying filesystem..."
msgstr "Mengesahkan sistem fail..."

#: ../tails_installer/creator.py:1029
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Tidak boleh mengubah label volum: %(message)s"

#: ../tails_installer/creator.py:1034 ../tails_installer/creator.py:1434
msgid "Installing bootloader..."
msgstr "Memasang pemuat but..."

#: ../tails_installer/creator.py:1061
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "Tidak dapat mencari modul COM32 '%s'"

#: ../tails_installer/creator.py:1069 ../tails_installer/creator.py:1452
#, python-format
msgid "Removing %(file)s"
msgstr "Membuang %(file)s"

#: ../tails_installer/creator.py:1183
#, python-format
msgid "%s already bootable"
msgstr "%s boleh dibutkan"

#: ../tails_installer/creator.py:1203
msgid "Unable to find partition"
msgstr "Gagal mencari sekatan"

#: ../tails_installer/creator.py:1226
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Memformat %(device)s sebagai FAT32"

#: ../tails_installer/creator.py:1286
msgid "Could not find syslinux' gptmbr.bin"
msgstr "Tidak dapat mencari syslinux' gptmbr.bin"

#: ../tails_installer/creator.py:1299
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "Membaca MBR terekstrak dari %s"

#: ../tails_installer/creator.py:1303
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1316 ../tails_installer/creator.py:1317
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Menetapkan semula Rekod But Master %s"

#: ../tails_installer/creator.py:1322
msgid "Drive is a loopback, skipping MBR reset"
msgstr ""

#: ../tails_installer/creator.py:1326 ../tails_installer/creator.py:1580
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Mengira SHA1 bagi %s"

#: ../tails_installer/creator.py:1351
msgid "Synchronizing data on disk..."
msgstr "Menyegerakkan data di dalam cakera..."

#: ../tails_installer/creator.py:1393
msgid "Error probing device"
msgstr "Ralat menguar peranti"

#: ../tails_installer/creator.py:1395
msgid "Unable to find any supported device"
msgstr "Tidak dapat mencari apa-apa peranti yang disokong"

#: ../tails_installer/creator.py:1405
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Pastikan pemacu USB anda dipalam dan telah diformat dengan sistem fail FAT"

#: ../tails_installer/creator.py:1408
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Fail sistem tidak disokong: %s\nSila sandar dan format pemacu USB anda dengan sistem fail FAT."

#: ../tails_installer/creator.py:1475
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr ""

#: ../tails_installer/creator.py:1528
msgid "Cannot find"
msgstr "Tidak dapat cari"

#: ../tails_installer/creator.py:1529
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Keluaran tidak diketahui : %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Memuat turun %s..."

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Ralat: Tidak dapat tetapkan label atau memperoleh UUID peranti anda. Gagal diteruskan."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Pemasangan selesai! (%s)"

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr "Pemasangan Tails gagal!"

#: ../tails_installer/gui.py:361
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Amaran: Alat ini perlu dijalankan dengan kelayakan Pentadbir. Untuk membuatnya, klik kanan pada ikon dan buka Properties. Di bawah tab Compatibility, tandakan pada kotak tanda \"Run this program as an administrator\"."

#: ../tails_installer/gui.py:373
msgid "Tails Installer"
msgstr "Pemasang Tails"

#: ../tails_installer/gui.py:423 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr "Klonkan Tails semasa"

#: ../tails_installer/gui.py:430 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr "Guna imej ISO Tail yang telah dimuat turun"

#: ../tails_installer/gui.py:467
msgid "Upgrade"
msgstr "Tatar"

#: ../tails_installer/gui.py:469
msgid "Manual Upgrade Instructions"
msgstr ""

#: ../tails_installer/gui.py:471
msgid "https://tails.boum.org/upgrade/"
msgstr ""

#: ../tails_installer/gui.py:479 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "Pasang"

#: ../tails_installer/gui.py:482 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr "Arahan Pemasangan"

#: ../tails_installer/gui.py:484
msgid "https://tails.boum.org/install/"
msgstr ""

#: ../tails_installer/gui.py:490
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "Peranti %(model)s %(vendor)s %(size)s (%(device)s)"

#: ../tails_installer/gui.py:502
msgid "No ISO image selected"
msgstr "Tiada imej ISO dipilih"

#: ../tails_installer/gui.py:503
msgid "Please select a Tails ISO image."
msgstr "Sila pilih satu imej ISO Tails."

#: ../tails_installer/gui.py:545
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:547
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:581
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:591
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr ""

#: ../tails_installer/gui.py:604
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr ""

#: ../tails_installer/gui.py:625
msgid "An error happened while installing Tails"
msgstr "Satu ralat berlaku ketika memasang Tails"

#: ../tails_installer/gui.py:637
msgid "Refreshing releases..."
msgstr "Menyegar semula keluaran..."

#: ../tails_installer/gui.py:642
msgid "Releases updated!"
msgstr "Keluaran dikemaskinikan!"

#: ../tails_installer/gui.py:695
msgid "Installation complete!"
msgstr "Pemasangan selesai!"

#: ../tails_installer/gui.py:744
msgid "Unable to mount device"
msgstr "Tidak dapat lekap peranti"

#: ../tails_installer/gui.py:751 ../tails_installer/gui.py:783
msgid "Confirm the target USB stick"
msgstr "Sahkan pena USB sasaran"

#: ../tails_installer/gui.py:752
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:770
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:778
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:779
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:821
msgid "Download complete!"
msgstr "Muat turun selesai!"

#: ../tails_installer/gui.py:825
msgid "Download failed: "
msgstr "Muat turun gagal:"

#: ../tails_installer/gui.py:826
msgid "You can try again to resume your download"
msgstr "Anda boleh cuba lagi untuk teruskan muat turun anda"

#: ../tails_installer/gui.py:834
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Fail terpilih tidak boleh dibaca. Sila betulkan keizinannya atau pilih fail yang lain."

#: ../tails_installer/gui.py:840
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Tidak boleh guna fail terpilih. Anda mungkin bernasib lebih baik jika anda alihkan ISO ke dalam direktori cakera keras anda (iaitu: C:\\)"

#: ../tails_installer/gui.py:846
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s dipilih"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' tidak wujud"

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr "'%s' bukan sebuah direktori"

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "Melangkau '%(filename)s'"

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "Terdapat masalah ketika melakukan `%s`.%s\n%s"

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr "Tidak dapat membuka peranti untuk penulisan."

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "Pilih satu distribusi untuk dimuat turun:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr "Pena USB sasaran:"

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr "Pasang semula (padam semua data)"
